packages/message/src/Collections.ts

```diff
@@ -21,29 +22,26 @@ const get = require('lodash/get');
  *
  * @alias module:Collections
  */
-function constructCollectionId(name, version) {
-  return `${name}___${version}`;
-}
+export const constructCollectionId = (name:string, version:string): string =>
+  `${name}___${version}`;
 
 /**
  * Get collection ID from execution message.
  *
- * @param {Object} message       - An execution message
+ * @param {Message.CumulusMessage} message - An execution message
```

Comment:
```suggestion
 * @param {CumulusMessage} message - An execution message
```

Reply:
The actual type is `Message.CumulusMessage` though?

Reply:
The type is `CumulusMessage`, ou just happen to be exporting it under `Message`. As it's currently defined, there is no namespace associated with that type.